src/libstrongswan/plugins/pkcs11/pkcs11_library.c

```diff
@@ -18,7 +18,13 @@
 
 #include "pkcs11_library.h"
 
-#include <dlfcn.h>
+#ifdef WIN32
+#  include <utils/compat/windows.h>
```

Comment:
In fact `<utils/compat/windows.h>` is not needed here, as above  `pkcs11_library.h` includes `<utils.h>`, but adding here for clarity.

Sorry for previous macros, I didn't notice compatibility layer in `<utils>`.